src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -80,29 +104,21 @@ function useCounterparty (agentId) {
   return { holofuelCounterparty, loading }
 }
 
-function useTransactionsWithCounterparties () {
-  const { data: { holofuelUser: whoami = {} } = {} } = useQuery(HolofuelUserQuery)
-  const { data: { holofuelInboxCounterparties = [] } = {} } = useQuery(HolofuelInboxCounterpartiesQuery, { fetchPolicy: 'cache-and-network' })
-  const { data: { holofuelActionableTransactions = [] } = {} } = useQuery(HolofuelActionableTransactionsQuery, { fetchPolicy: 'cache-and-network' })
-  const { data: { holofuelNonPendingTransactions = [] } = {} } = useQuery(HolofuelNonPendingTransactionsQuery, { fetchPolicy: 'cache-and-network' })
-
-  const updateCounterparties = (transactions, counterparties) => transactions.map(transaction => ({
```

Comment:
👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 

Very happy that this is gone.

We should make a chore to do the same thing for the other Pages.